docs/storage.rst

```diff
@@ -117,8 +121,9 @@ Once installed the PostgreSQL server must be configured. Run the commands
 
 .. code-block:: console
 
-  sudo postgresql-setup initdb
-  sudo systemctl start postgresql
+  sudo /usr/pgsql-9.6/bin/postgresql96-setup initdb
+  sudo systemctl enable postgresql-9.6
+  sudo systemctl start postgresql-9.6
   sudo -u postgres createuser -d fledge
```

Comment:
Here we require to create user of the username for example centos (more generically output of whoami command), because when I tried to start fledge after changing db to postgres it gave following error:
```
[centos@ip-10-0-0-151 ~]$ /usr/local/fledge/bin/fledge start
psql: FATAL:  role "centos" does not exist
```
Please have a look.